index.js

```diff
@@ -63,10 +66,49 @@ function toggleColor(element) {
     if (element.style.color === "green") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function() {
+    toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector('button.like-button')
+
+button.addEventListener('click', function() {
+    // const likesPtag = document.querySelector('p.likes')
+
+    let currLikes = parseInt(likes.textContent)
+    likes.textContent = `${currLikes + 1} Likes`
+
+})
+
+
 /***** Deliverable 3 *****/
+
+const newSightingForm = document.querySelector('form#new-animal-sighting-form')
+
+newSightingForm.addEventListener('submit', function (event) {
+    event.preventDefault()
+    const species = event.target[0].value
+    const video = event.target[1].value
+    const photo = event.target[2].value
+    const description = event.target[3].value
+
+    const lastIndex = traveler.animalSightings.length - 1
+    const newAnimalSightingObject = {
+        id: traveler.animalSightings[lastIndex].id + 1,
+        travelerId: 1,
+        species: species,
+        photo: photo,
+        link: video,
+        description: description
+    }
+        
```

Comment:
We could also include something similar to this functionality for this problem. We could include:
`traveler.animalSightings.push(newSightingObject)`
In fact, it's a good idea to include this line because of the way we're creating custom IDs. If we want future animal sightings to have a unique ID, then we have to push all new animal sighting objects into the array.